### PR TITLE
fix: audit string and variables lectures quizzes reviews

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

Changed the order of two videos
Changed 5 questions in the quiz to topics that were covered in the preceeding lectures
Fixed some comprehension questions
Some typos here and there

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md [hunk 1]
```diff
@@ -109,11 +109,11 @@ What is the difference between `let` and `const` variable declarations in JavaSc
 
 #### --distractors--
 
-`let` is block scoped, `const` is not.
+
 
 ---
 
-`const` is block scoped, `let` is not.
+
```
Comment (a2937) [inline curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md:116]: How about a distractor related to `hoisting`?

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md [hunk 2]
```diff
@@ -127,23 +127,23 @@ What is the difference between `let` and `const` variable declarations in JavaSc
 
 #### --text--
 
-What is the difference between local and global scope in JavaScript?
+
```
Comment (a2937) [inline curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md:130]: Maybe a question pertaining all three different scopes?

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md [hunk 3]
```diff
@@ -428,79 +418,65 @@ Which of the following is NOT a valid string concatenation method in JavaScript?
...
 #### --answer--
 
-`ReferenceError: world is not defined`
+
 
 ### --question--
 
 #### --text--
 
-What will be the output of the code below:
```
Comment (a2937) [inline curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md:461]: Maybe we can have a question about camel casing here?

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md (hunk 1 above)
Comment (ilenia-magoni) [reply curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md:116]: I don't remember hoisting mentioned in the lectures

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md (hunk 2 above)
Comment (ilenia-magoni) [reply curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md:130]: scopes are not mentioned in the lectures

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md [hunk 4]
```diff
@@ -428,79 +418,65 @@ Which of the following is NOT a valid string concatenation method in JavaScript?
 
 #### --text--
 
-What will be the output of the following code:
```
Comment (a2937) [inline curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md:431]: Maybe a question about which variable name is a valid identifier?

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md (hunk 3 above)
Comment (ilenia-magoni) [reply curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md:461]: like `Which of the following variable names uses camelCase correctly?`?

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md (hunk 4 above)
Comment (ilenia-magoni) [reply curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md:431]: good idea, those questions are always good

File: curriculum/challenges/english/25-front-end-development/lecture-introduction-to-javascript/672d49c4e899345f5b33c24c.md [hunk 5]
```diff
@@ -110,11 +110,11 @@ Think about the basic structure of `console.log` and where the variable should g
 
 ---
 
-`console.write(age);`
+`console.log("age");`
 
 ### --feedback--
 
-Think about the basic structure of `console.log` and where the variable should go.
+Think about what is the difference between a variable and a string.
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/lecture-introduction-to-javascript/672d49c4e899345f5b33c24c.md:117]:
```suggestion
Think about the difference between a variable and a string.
```

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md (hunk 3 above)
Comment (a2937) [reply curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md:461]: Yeah

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md [hunk 6]
```diff
@@ -428,79 +424,65 @@ Which of the following is NOT a valid string concatenation method in JavaScript?
...
 
-`TypeError: Assignment to constant variable.`
+`;`
 
 ### --question--
 
 #### --text--
 
-Given the string `freeCodeLamp`, which JavaScript string method can be used to turn it into `freeCodeCamp`?
+Which of the following is NOT a data type in JavaScript?
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md:471]:
```suggestion
Which of the following is NOT a JavaScript data type?
```

**Review by Dario-DC — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/lecture-working-with-strings-in-javascript/673263e80dd43da7df3ae565.md
Comment (gikf) [inline curriculum/challenges/english/25-front-end-development/lecture-working-with-strings-in-javascript/673263e80dd43da7df3ae565.md:1]: Could you add backticks to the numbers in options for the first question?

**Review by a2937 — APPROVED** [review]

**Review by zairahira — APPROVED** [review]

**Pull request merged**